ledger/acctupdates.go

```diff
@@ -60,6 +62,8 @@ const (
 	// trieAccumulatedChangesFlush defines the number of pending changes that would be applied to the merkle trie before
 	// we attempt to commit them to disk while writing a batch of rounds balances to disk.
 	trieAccumulatedChangesFlush = 256
+	// directory name in which all the catchpoints files are stored
+	catchpointDirName = "catchpoints"
```

Comment:
It's great that you added this. I wasn't aware that we have multiple instances of that string in the codebase. However, please also make sure you're replacing the string with that constant uniformly - there are additional places in the _test files that use it. look in acctupdates_test.go and catchpointwriter_test.go

Reply:
Done